lucene/highlighter/src/java/org/apache/lucene/search/uhighlight/Passage.java

```diff
@@ -193,8 +203,4 @@ public void setEndOffset(int endOffset) {
     this.endOffset = endOffset;
   }
 
-  /** @lucene.internal */
-  public void setScore(float score) {
```

Comment:
We should probably keep setScore(float) for users who want to build scores their own way.